### ISSUE TITLE
[A11Y] Add "aria-disabled" attribute

### ISSUE THREAD

**Issue by ValeryKrukovich** [react-component/collapse#229]
For accessibility, collapse element should have an aria-disabled attribute

Cases: aria-disabled=

- `true` collapse is disabled
- `false` collapse is not disabled

There are elements that use color as the only visual means to convey state, selection, or other information. 

Example:
The "Panel header" accordion is styled to look disabled but it is not conveyed programmatically. 

Users who cannot distinguish colors will not have access to the information conveyed by the use of color and may find the page unusable.

learn more - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled

**Comment by afc163** [comment]
close https://github.com/react-component/collapse/pull/230
